Html and css revision

Safkat-Khan opened the pull request:
Revised in html and css for web tech

WasikAhmed reviewed (commented):
Thank you for your work on this PR! I have a few suggestions to enhance the readability and maintainability of the code.

- Please add comments throughout the code to explain and clarify. This will help others quickly understand the intent behind each part.
- Review the file organization.
- Maintain consistent formatting for readability.

Once updated, the changes will greatly improve the clarity and accessibility of the code for future maintenance. Let me know if you have any questions!

Safkat-Khan commented:
Thank you for your review! I've implemented your suggestions,

Implemented your suggestions: 
-  Added comments for clarity
-  Reorganized file structure
-  Enhanced formatting for readability

Safkat-Khan commented:
Thank you for your review! I've implemented your suggestions,

Implemented your suggestions:
- Added comments for clarity
- Reorganized file structure
- Enhanced formatting for readability
